RetroBar/Languages/català.xaml

```diff
@@ -19,24 +20,84 @@
         <s:String>Dreta</s:String>
         <s:String>Inferior</s:String>
     </x:Array>
+    <s:String x:Key="rowcount_text">Nombre de _files:</s:String>
+    <s:String x:Key="rowcount_tip">El nombre de files a la barra de tasques quan es trobi a la part superior o inferior.</s:String>
+    <s:String x:Key="taskbar_width">_Amplada de la barra de tasques:</s:String>
     <s:String x:Key="allow_font_smoothing">Permet el _suavitzat dels tipus de lletra</s:String>
-    <s:String x:Key="collapse_tray_icons">_Oculta les icones de notificació</s:String>
+    <s:String x:Key="collapse_tray_icons">Oculta les icones de _notificació</s:String>
     <s:String x:Key="customize">_Personalització...</s:String>
+    <s:String x:Key="show_input_language">Mostra la _llengua d'entrada</s:String>
     <s:String x:Key="show_clock">Mostra el _rellotge</s:String>
     <s:String x:Key="show_multi_mon">Mostra en _múltiples pantalles</s:String>
+    <s:String x:Key="show_start_button_multi_mon">Mostra el botó Inicia en t_otes les barres de tasques</s:String>
     <s:String x:Key="show_quick_launch">Mostra l'_inici ràpid</s:String>
     <s:String x:Key="select_location">_Tria ubicació...</s:String>
-    <s:String x:Key="quick_launch_folder">Inici ràpid: tria una carpeta</s:String>
+    <s:String x:Key="quick_launch_folder">Inici ràpid: trieu una carpeta</s:String>
```

Comment:
Per què fer aquest canvi? We require the use of a formal "classic" and maybe even "traditional" language unless it is referenced in the [official translations of the system and applications](https://msit.powerbi.com/view?r=eyJrIjoiMmE2NjJhMDMtNTY3MC00MmI2LWFmOWUtYWM5YTVjODI5MjQwIiwidCI6IjcyZjk4OGJmLTg2ZjEtNDFhZi05MWFiLTJkN2NkMDExZGI0NyIsImMiOjV9).
We can skip some of the [modern style guidelines](https://aka.ms/catalan-styleguide).